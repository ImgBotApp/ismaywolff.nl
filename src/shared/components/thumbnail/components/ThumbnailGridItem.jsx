import styled from 'styled-components'

const ThumbnailGridItem = styled.div`
  grid-column: span 2;
  position: relative;

  @media (min-width: 30em) {
    grid-column: span 1;
  }
`

export default ThumbnailGridItem
