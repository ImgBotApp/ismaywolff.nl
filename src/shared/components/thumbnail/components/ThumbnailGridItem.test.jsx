/**
 * @jest-environment node
 */

import 'jest-styled-components'
import React from 'react'
import { shallow } from 'enzyme'
import ThumbnailGridItem from './ThumbnailGridItem'

describe('<ThumbnailGridItem />', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<ThumbnailGridItem />)
    expect(wrapper).toMatchStyledComponentsSnapshot()
  })
})
