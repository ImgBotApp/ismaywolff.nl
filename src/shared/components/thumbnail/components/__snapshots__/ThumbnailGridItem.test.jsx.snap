// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ThumbnailGridItem /> renders correctly 1`] = `
.c0 {
  box-shadow: var(--box-shadow);
  grid-column: span 2;
  position: relative;
}

@media (min-width: 30em) {
  .c0 {
    grid-column: span 1;
  }
}

<div
  className="c0"
/>
`;
